XCUITests/FxScreenGraph.swift

```diff
@@ -990,10 +991,14 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.tap(app.buttons["HomePanels.Downloads"], to: HomePanel_Downloads)
     }
 
+    map.addScreenState(LoginsBrowserTab) { screenState in
+    	screenState.backAction = navigationControllerBackAction
+    }
+
     map.addScreenState(BrowserTabMenu) { screenState in
         screenState.tap(app.tables.cells["menu-Settings"], to: SettingsScreen)
-
         screenState.tap(app.tables.cells["menu-sync"], to: FxASigninScreen, if: "fxaUsername == nil")
+        screenState.tap(app.tables.cells["Logins & Passwords"], to: LoginsBrowserTab)
```

Comment:
Looks like there is an id for this, "key" not sure why that's the id...not very meaningful...if it is not very difficult we could try to change it something more related to the option...